docs/docs/adding-pagination.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Adding pagination
+---
+
+A page displaying a list of content gets longer as the amount of content grows.
+Pagination is the technique of spreading that content across multiple pages.
+
+The goal is to create multiple pages (from a single [template](/docs/building-with-components/#page-template-components)) that show a limited number of items.
+
+Each page will [query GraphQL](/docs/querying-with-graphql/) for those specific items.
+
+The information needed to query for those specific items (a [limit](/docs/graphql-reference/#limit) and [skip](/docs/graphql-reference/#skip) value) will come from the [`context`](/docs/graphql-reference/#query-variables) that is added when [creating pages](/docs/creating-and-modifying-pages/#creating-pages-in-gatsby-nodejs) in `gatsby-node`
+
+### Example
+
+```js{20-25}:title=src/templates/blog-list-template.js
```

Comment:
Nice job adding this 🎉

Reply:
💯 Just thought the same scrolling through this!
Also totally forgot seconding everyone elses comments in here when I left a comment earlier — great work @NickyMeuleman! 🙏 

Btw. @DSchau, I just remembered we have a https://github.com/gatsbyjs/gatsby/blob/master/www/src/utils/copy-to-clipboard.js in here already… ;-) (was used to copy a starter URL to the clipboard via a button on a starter card, which got removed).